modules/liveIntentIdSystem.js

```diff
@@ -100,10 +100,11 @@ export const liveIntentIdSubmodule = {
    * `publisherId` params.
    * @function
    * @param {{unifiedId:string}} value
-   * @param {SubmoduleParams|undefined} [configParams]
+   * @param {SubmoduleParams|undefined} [config]
```

Comment:
Here `SubmoduleConfig` instead of `SubmoduleParams`?

Reply:
yep